modules/atomic-widgets/props-resolver/transformers/styles/background-image-overlay-transformer.php

```diff
@@ -26,7 +26,7 @@ public function transform( $value, $key ) {
 			$background_style .= ' ' . $position_and_size_style;
 		}
 
-		return $background_style;
+		return $background_style . ( $value['repeat'] ? ' ' . $value['repeat'] : '' );
```

Comment:
@nami-p 

Can you please create a separate if statement for this and keep the same order as we have in the packages PR (repeat and attachment before position and size).